Makefile

```diff
@@ -1,11 +1,11 @@
 
 BUILD_DIR=build
-COVERAGE_DIR=${BUILD_DIR}/coverage
+COVERAGE_DIR=$(BUILD_DIR)/coverage
```

Comment:
I'm very happy to have it unified, but lets do it in a different PR. Could you open an other one for that?